src/www/ui/template/advice_license-edit.html.twig

```diff
@@ -12,6 +12,24 @@
   <input type="hidden" name="mod" value="advice_license"/>
   <input type="hidden" name="rf" value="{{ rf_pk }}"/>
   <table>
+  {% if rf_lastmodified is not empty %}
+    <tr>
+      <td>{{ 'Username Created'|trans }}:</td>
```

Comment:
Just thinking, "User Created" is better than "Username Created" while displaying in UI. What do you think?

Reply:
yeah